src/org.ts

```diff
@@ -540,6 +532,66 @@ export class Org extends AsyncCreatable<Org.Options> {
       }
     });
   }
+
+  /**
+   * Remove the org users auth file.
+   * @param throwWhenRemoveFails true if manageDelete should throw or not if the deleted fails.
+   */
+  private async removeUsers(throwWhenRemoveFails: boolean) {
+    this.logger.debug(`Removing users associate with org: ${this.getOrgId()}`);
+    const config = await this.retrieveOrgUsersConfig();
+    this.logger.debug(`using path for org users: ${config.getPath()}`);
+    if (await config.exists()) {
+      const _auths: AuthInfo[] = await this.readUserAuthFiles();
+      const aliases: Aliases = await Aliases.create(Aliases.getDefaultOptions());
+      this.logger.info(`Cleaning up usernames in org: ${this.getOrgId()}`);
+
+      for (const auth of _auths) {
+        const username = auth.getFields().username;
+
+        const aliasKeys = (username && aliases.getKeysByValue(username)) || [];
+        aliases.unsetAll(aliasKeys);
+
+        let orgForUser;
+        if (username === this.getUsername()) {
+          orgForUser = this;
+        } else {
+          const _info = await AuthInfo.create({ username });
+          const connection: Connection = await Connection.create({ authInfo: _info });
+          orgForUser = await Org.create({ connection });
+        }
+
+        const orgType = this.isDevHubOrg() ? Config.DEFAULT_DEV_HUB_USERNAME : Config.DEFAULT_USERNAME;
```

Comment:
It's been a while since I've been through this code.  What is done differently for devhub vs 'regular' org?  We now have another org type with sandbox.

Reply:
It's cleaning up configs. For example, if the org in question is configured as the targetusername or defaultdevhubusername.

Reply:
👍